### ISSUE TITLE
Freefem v4.10 on apple m1 does not run

### ISSUE THREAD

**Issue by cpraveen** [FreeFem/FreeFem-sources#217]
When I run the M1 version of 4.10, it says file is damaged. 

When I check "Get info" on the freefem application, it actually says it is Intel.

Has anybody succeeded in running the M1 version of 4.10 ?

<img width="328" alt="Screenshot 2022-02-06 at 8 23 10 AM" src="https://user-images.githubusercontent.com/748769/152666030-e7c6c4fe-5889-4f69-9f2e-93b9511068f4.png">

**Comment by prj-** [comment]
> Has anybody succeeded in running the M1 version of 4.10 ?

Yes.

Are you sure you are using the proper `.dmg`? You could compile from sources otherwise.

**Comment by cpraveen** [comment]
I download this file

https://github.com/FreeFem/FreeFem-sources/releases/download/v4.10/FreeFem++-4.10-Apple-M1.zip

and it does not work.

**Comment by frederichecht** [comment]
Yes , I have correct the version for the download  just do on thermal in case of error : 

sudo xattr -rc /Applications/FreeFem++.app/

otherwise I a put a develop  version  here

FreeFem++-4.10-52-gab8be280-Apple-M1.zip <https://www.ljll.math.upmc.fr/hecht/ftp/tmp/FreeFem++-4.10-52-gab8be280-Apple-M1.zip>

Best Regards, 

Frédéric Hecht.   

-----------------------------------------------------------------------------------------------------------------------------------
Laboratoire  Jacques-Louis Lions,                UPMC Sorbonne Université
BC187,  4 Place  Jussieu,     75252  PARIS cedex 05,               France
Campus Jussieu,       Barre 15-25,           3 etage           Bureau 307 
Projet Alpines ,   Inria de Paris, 2 rue Simone Iff Voie DQ12 75012 Paris
tel:  +33 1 44274411,        mob: +33 6 62198986,     fax: +33 1 44277200
***@***.*** https://www.ljll.math.upmc.fr/hecht
software:  FreeFem++          web site:       http://www.freefem.org/ff++

**Comment by frederichecht** [comment]
Dear Yes sorry, for that,

the correction is: 

sudo xattr -rc /Applications/FreeFem++.app/

otherwise I a put a ne develop  version  here

FreeFem++-4.10-52-gab8be280-Apple-M1.zip <https://www.ljll.math.upmc.fr/hecht/ftp/tmp/FreeFem++-4.10-52-gab8be280-Apple-M1.zip>



Best Regards, 

Frédéric Hecht.   

-----------------------------------------------------------------------------------------------------------------------------------
Laboratoire  Jacques-Louis Lions,                UPMC Sorbonne Université
BC187,  4 Place  Jussieu,     75252  PARIS cedex 05,               France
Campus Jussieu,       Barre 15-25,           3 etage           Bureau 307 
Projet Alpines ,   Inria de Paris, 2 rue Simone Iff Voie DQ12 75012 Paris
tel:  +33 1 44274411,        mob: +33 6 62198986,     fax: +33 1 44277200
***@***.*** https://www.ljll.math.upmc.fr/hecht
software:  FreeFem++          web site:       http://www.freefem.org/ff++
